docs/UserGuide.md

```diff
@@ -200,6 +200,38 @@ If `/edit` is supplied instead of a note, (e.g. `note 1 /edit`), the contents of
 > Executing the command `note 1 /edit` will replace the contents of the command box with `note 1 Wants to gain muscle`.
 <hr>
 
+### Adding or overriding exercise(s) of clients : `fitadd`
+
+Format: `fitadd INDEX [n/EXERCISE_NAME] [s/SETS] [r/REPS] [b/BREAK_BETWEEN_SETS_IN_SECONDS] [/arms] [/legs] [/chest] [/back] [/shoulders] [/abs] [/all] `
+
+* Adds the specified exercise(s) to the client specified by `INDEX`. The index refers to the index number shown in the displayed client list. The index **must be a positive integer** 1, 2, 3, …​
+* If an exercise with same name already exists for the client, the old exercise will be overwritten with the newly supplied field values, or a default set of values if not supplied.
```

Comment:
Might be good to mention the default values.

Reply:
Default values are mentioned within the format.

Reply:
Added!